extensions/resteasy-jsonb/deployment/src/main/java/io/quarkus/resteasy/jsonb/deployment/JsonbConfig.java

```diff
@@ -0,0 +1,72 @@
+package io.quarkus.resteasy.jsonb.deployment;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_TIME;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.json.bind.config.PropertyOrderStrategy;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = BUILD_TIME)
+public class JsonbConfig {
+
+    static final Set<String> ALLOWED_PROPERTY_ORDER_VALUES = Collections.unmodifiableSet(
+            new HashSet<>(Arrays.asList(PropertyOrderStrategy.LEXICOGRAPHICAL, PropertyOrderStrategy.ANY,
+                    PropertyOrderStrategy.REVERSE)));
+
+    /**
+     * If enabled, Quarkus will a create a JAX-RS resolves that configures JSON-B with the properties
+     * specified here
+     * It will also attempt to generate serializers for JAX-RS return types
+     */
+    @ConfigItem(defaultValue = "false")
+    boolean enabled;
```

Comment:
I have elected to have the feature off by default - I'm not sure it's the correct choice so I am open to suggestions

Reply:
IMHO the configuration of JSON-B is a long waiting functionality and cannot cause any regression if the default are the Yasson one so it should always apply.
The boolean should only apply to the generation of serializers, this could be disabled by default (features in BETA) and enabled after some feedback from early adopters

Reply:
I pondered that option myself as well. Honestly I think what you say makes a lot of sense and I wil end up doing it unless others object